src/test/java/com/snowflake/kafka/connector/internal/ConnectionServiceIT.java

```diff
@@ -56,6 +57,8 @@ public void testEncryptedKey() {
   }
 
   @Test
+  @Disabled
+  // OAuth tests are temporary disabled
```

Comment:
Junit's disabled accepts reason as a parameter. Then this reason can be displayed in IDE and other tools
```java
@Disabled("OAuth tests are temporary disabled")
```

Reply:
Thx, done